config/install/pathauto.pattern.user_profile_urls.yml

```diff
@@ -0,0 +1,13 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - user
+id: user_profile_urls
+label: 'User Profile URLs'
+type: 'canonical_entities:user'
+pattern: 'user/[user:account-name]'
+selection_criteria: {  }
+selection_logic: and
+weight: 0
```

Comment:
We, and possibly other projects, have implemented our own user path settings, and the webspark.install code that loads in this new pathauto pattern creates a competing rule that we want to ensure doesn't take precedence over existing rules.  

As per the behaviors you see depicted regarding pathauto weight in configs in [this Drupal.org issue](https://www.drupal.org/project/pathauto/issues/2839643), can you set the weight here to 5 or something, so if an existing  rule with weight 0 exists, we don't have unexpected tie-breaker behavior?